lib/hexpm/accounts/organization.ex

```diff
@@ -64,11 +62,19 @@ defmodule Hexpm.Accounts.Organization do
   def role_or_higher("write"), do: ["write", "admin"]
   def role_or_higher("admin"), do: ["admin"]
 
-  def hexpm() do
+  def hexpm(opts \\ []) do
```

Comment:
`Organization.hexpm()` feels a bit awkward, to me conceptually there's no such thing. Is this because repository belongs to an organization? I don't really have a better solution to this at the moment, just thought I'll mention it anyway.

Reply:
I agree, but we would need a lot of special checks for the hexpm repository check since we assume all other repositories have an organization.